src/constants/fontTheme.ts

```diff
@@ -0,0 +1,85 @@
+interface FontStyle {
+  fontFamily: string;
+  fontWeight:
+    | 'normal'
+    | 'bold'
+    | '100'
+    | '200'
+    | '300'
+    | '400'
+    | '500'
+    | '600'
+    | '700'
+    | '800'
+    | '900';
+}
+
+interface FontTheme {
+  fontSizes: {
+    header1: { size: number; lineHeight: string | 'auto'; fontFamily: string };
+    header2: { size: number; lineHeight: string | 'auto'; fontFamily: string };
+    subtitle1: {
+      size: number;
+      lineHeight: string | 'auto';
+      fontFamily: string;
+    };
+    body1: { size: number; lineHeight: string | 'auto'; fontFamily: string };
+    buttonLarge: {
+      size: number;
+      lineHeight: string | 'auto';
+      fontFamily: string;
+    };
+    buttonSmall: {
+      size: number;
+      lineHeight: string | 'auto';
+      fontFamily: string;
+    };
+  };
+  fonts: {
+    regular: FontStyle;
+    bold: FontStyle;
+    medium: FontStyle;
+    light: FontStyle;
+    heavy: FontStyle;
+  };
+}
+
+const fontTheme: FontTheme = {
+  fontSizes: {
+    header1: {
+      size: 24,
+      lineHeight: 'auto',
+      fontFamily: 'AppleSDGothicNeoB00',
+    },
```

Comment:
수고하셨습니다! 몇 가지 제안드리겠습니다.
FontTheme 사용할 때, fontTheme.header1처럼 직관적으로 접근해서 사용할 수 있으면 좋을 것 같습니다!
그리고 피그마 참고해서 lineHeight값도 지정해두면 좋을 것 같아요

```suggestion
const fontTheme: FontTheme = {
  header1: {
    fontFamily: 'AppleSDGothicNeoB00',
    fontWeight: 400,
    size: 24,
    lineHeight: 33.46,
  },
```

Reply:
참고해서 lineHeight값도 지정하도록 하겠습니다!  fontFamily랑, fontSize를 따로 지정하는 방식보다는, 스타일 가이드에 있는 것들은 직관적으로 fontTheme.header1 과 같이 직관적으로 개발하면서, 각각 지정해야하는 경우에는, 또 각각 지정할 수 있도록 수정해보겠습니다!